core-bundle/src/Resources/config/controller.yml

```diff
@@ -0,0 +1,11 @@
+services:
+    _defaults:
+        autoconfigure: true
+        public: true
+
+    Contao\CoreBundle\Controller\Page\RootPageController:
```

Comment:
Why did you not move the other controller definitions here as well?

Reply:
I didn't want to move all the "old" and existing stuff, I intended to add all new page controllers here.

Reply:
This will be totally inconsistent then. Either move all controller definitions here or add this definition where the other controller definitions are. 🤷‍♂️